ee/clickhouse/queries/funnels/test/__snapshots__/test_funnel.ambr

```diff
@@ -63,6 +67,10 @@
                       (SELECT id
                        FROM person
                        WHERE team_id = 2
+                         AND id IN
```

Comment:
quick nit: We should only do this when a property exists, as otherwise there's no reduction in size

Reply:
good catch